src/main/kotlin/com/example/minibank/config/FTProperties.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.minibank.config
+
+import org.springframework.boot.context.properties.ConfigurationProperties
+import kotlin.properties.Delegates
+
+@Suppress("ConfigurationProperties")
+@ConfigurationProperties
+class FTProperties {
+
```

Comment:
🚫 **[ktlint]** <standard:no-empty-first-line-in-class-body> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Class body should not start with blank line